### PR TITLE
Mvp 1602/sdk example

### PR DESCRIPTION
**Pull request by zain-notifi**

### CODE REVIEW

**Comment by zain-notifi** [comment]
will update the readme later today

File: packages/notifi-react-example/public/robots.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (Nathan-Notifi) [inline packages/notifi-react-example/public/robots.txt:1]: What does this do?

Comment (zain-notifi) [reply packages/notifi-react-example/public/robots.txt:1]: comes default with create-react-app. I think it helps block web crawlers. We can remove if unnecessary!

Comment (Nathan-Notifi) [reply packages/notifi-react-example/public/robots.txt:1]: i think that's fine!

**Review by juni-notifi — APPROVED** [review]
Nice, LGTM

I kinda wish we had gone with React 18 instead, but we can take that on incrementally.

For folks using this: Remember to run `npx lerna run build` for the example repo to be able to pick up your local changes!

**Pull request merged**